### PR TITLE
CHANGELOG for 0.27.2

### PR DESCRIPTION
**Pull request by datho7561**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [0.27.2](https://github.com/redhat-developer/vscode-xml/milestone/38?closed=1) (November 25, 2024)
+
+### Bug Fixes
+
+ * Fix support for working with XML files on shared folders. See [#1033](https://github.com/redhat-developer/vscode-xml/issues/1033).
+ * Fix interaction between XInclude and `<?xml-model>` PI. See [#1021](https://github.com/redhat-developer/vscode-xml/issues/1021).
+ * Support xsi:type overrides when resolving content model. See [eclipse-lemminx/lemminx#1647](https://github.com/eclipse-lemminx/lemminx/pull/1647).
```
Comment (rgrunber) [inline CHANGELOG.md:9]: Is it worth also making it `XInclude`, and `xsi:type` ?

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "SVG"
   ],
   "xmlServer": {
-    "version": "0.28.0"
+    "version": "0.29.0"
```
[the remark this answers is not in the document]
Comment (datho7561) [reply package.json:25]: The latest release of lemminx is 0.29.0. The version numbers between vscode-xml and lemminx are no longer synchronized.

[the remark this answers is not in the document]
Comment (rgrunber) [reply package.json:25]: Whoops, you're right, it's 0.29.0. Alright.

**Comment by datho7561** [comment]
Thanks for the review, Roland!

**Pull request merged**